src/shared/ui/button/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import styles from './button.module.css';
+import React from "react";
+
+type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+  size?: '' | 'semi' | 'large';
```

Comment:
usually 'xs', 'sm', 'md', 'lg'
remove empty string and add default value in props

Reply:
removed & added